docs/ResultsList.md

```diff
@@ -0,0 +1,45 @@
+# Search Bar
```

Comment:
Change the README to follow Breadcrumb README model:
https://github.com/vtex-apps/breadcrumb